pyxis/find_signatures.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+"""
+Python script to find all references for a given repository and manifest_digest
```

Comment:
I'm missing the keyword "signature" in this description ;)

So something like this maybe?
```suggestion
Python script to find all signatures in Pyxis for a given repository and manifest_digest 
and save a list of references that are signed in a text file, one reference per line
```

Reply:
thanks. added your suggestion.